### DISCUSSION TITLE
Open a new URL in current tab

### DISCUSSION THREAD

**Discussion by madjxatw** [brookhong/Surfingkeys#1957] [Q&A] [answered]
Hi, I typed `go` to attempt to open a new URL in the current tab but got a google search page taking my URL as keyword. I tried `t` and it does open my URL but in a new tab. How can I achieve what I exactly want?

**Comment by GrewHair** [comment, marked as answer]
Tried it with my setup - and it seems you have to either type out `https://` at the beginning of your URL, or pick a candidate from the list using arrows, then press enter. It doesn't auto-add this scheme prefix as most browser omnibars do.

As a workaround, you may add `settings.focusFirstCandidate = true` to your config. This way it will focus the candidate for you, so you just have to press enter. Not sure tho whether it will always pick the right site as the first candidate - but it works for me.

Btw, after pressing <kbd>t</kbd>, if you want your page opened in the current tab instead of a new one - press <kbd>Shift-Enter</kbd> instead of just <kbd>Enter</kbd>.
